public/i18n/en/translation.json

```diff
@@ -777,7 +777,15 @@
       "vpn_ip_address": "VPN IP address",
       "disconnect": "Disconnect",
       "disconnect_unit_message": "The unit will be disconnected from the controller and will no longer be accessible from the controller itself. Do you want to proceed?",
-      "data_updated_every_seconds": "Data updated every {seconds} seconds"
+      "data_updated_every_seconds": "Data updated every {seconds} seconds",
+      "disclaimer": "Disclaimer",
+      "connect_unit_message": "Unit logs will be transmitted to the Controller for the purposes of storage, monitoring, and analysis of network usage. \nIf you do not wish for this data to leave the device, do not connect to the Controller.",
+      "confirm_and_connect": "Confirm and connect",
+      "sending_data_to_controller": "Sending Monitoring data to Controller",
+      "sending_data_to_controller_tooltip": "Complete monitoring available on Grafana with NethSecurity",
```

Comment:
Let's use [linked messages](https://vue-i18n.intlify.dev/guide/essentials/syntax#linked-messages) to build the string with the link to subscription: different languages might use a different position for the word "subscription".

Reply:
Fine by me, but how do we set the link for a translated variable in the tooltip?

Reply:
Sorry I linked the wrong docs page, check this out: https://vue-i18n.intlify.dev/guide/advanced/component.html#slots-syntax-usage

You can also find other usage examples by searching `<i18n-t` inside the repo source

Reply:
Never used the component, very nice to have!